de.fu_berlin.inf.dpp.ui.frontend/META-INF/MANIFEST.MF

```diff
@@ -5,4 +5,3 @@ Bundle-SymbolicName: de.fu_berlin.inf.dpp.ui.frontend
 Bundle-Version: 0.1.0.DEVEL
 Bundle-Vendor: FU-Berlin
 Fragment-Host: de.fu_berlin.inf.dpp.ui
-Bundle-RequiredExecutionEnvironment: JavaSE-1.6
```

Comment:
This was the case in a former version of my PR, but now (because of the MANIFEST parsing and so on) this manifest is used. I will change the line to JavaSE-1.8